x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -7,41 +7,78 @@
 
 import path from 'path';
 
-export const paths = {
-  archivesPath: path.resolve(__dirname, '../../../../../../.chromium'),
-  baseUrl: 'https://storage.googleapis.com/headless_shell/',
-  packages: [
+// We download zip files from a Kibana team GCS bucket named `headless_shell`
```

Comment:
This file was almost entirely re-written because there is a lot of code that uses the object which had no abstractions available to work with the structure of the data.